demo-app/build.gradle.kts

```diff
@@ -0,0 +1,88 @@
+import java.io.FileInputStream
+import java.util.Properties
+
+plugins {
+    id("otel.android-app-conventions")
+}
+
+val localProperties = Properties()
+localProperties.load(FileInputStream(rootProject.file("local.properties")))
+
+android {
+    namespace = "io.opentelemetry.android.demo"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "io.opentelemetry.android.demo"
+        minSdk = 21
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        all {
+            val accessToken = localProperties["rum.access.token"] as String?
+            resValue("string", "rum_access_token", accessToken ?: "fakebroken")
+        }
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro",
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+        isCoreLibraryDesugaringEnabled = true
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+    buildFeatures {
+        compose = true
+        viewBinding = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.5.13"
+    }
+    packaging {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
```

Comment:
Do we need this line?

Reply:
Not sure, probably not, it was auto-generated I'm sure. I'll try removing it.